### PR TITLE
gemify taskqueue

### PR DESCRIPTION
**Pull request by snatchev**
This PR makes the repo into a gem, able to be pulled into another project via bundler.

https://github.com/fastlane/ci/pull/345

This gem is not intended to be publicly distributed yet.

### CODE REVIEW

File: taskqueue.gemspec [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  s.name        = 'taskqueue'
+  s.version     = '0.0.0'
+  s.date        = '2018-03-20'
+  s.summary     = "Hola!"
+  s.description = "A simple hello world gem"
+  s.authors     = ["Joshua Liebowitz"]
+  s.email       = 'taquitos@gmail.com'
+  s.files       = ["lib/task.rb", "lib/task_queue.rb", "lib/queue_worker.rb", "lib/recreatable_task.rb"]
+  s.homepage    =
+    'http://rubygems.org/gems/hola'
```
Comment (minuscorp) [inline taskqueue.gemspec]: wat 🤣

Comment (snatchev) [reply taskqueue.gemspec]: copy pasta strikes again. fixed.

File: Gemfile [hunk 2]
```diff
@@ -1,8 +1,3 @@
 source('https://rubygems.org')
 
-group :development do
-  gem 'pry'
-  gem 'pry-byebug'
-  gem 'rspec'
-  gem 'rubocop'
-end
+gemspec
```
Comment (KrauseFx) [inline Gemfile]: missing new line

**Review by taquitos — APPROVED** [review]
👍:goat:

**Pull request merged**